index.html

```diff
@@ -1,136 +1,164 @@
 <!DOCTYPE html>
-<html lang='en'>
-  <head>
-    <meta charset='UTF-8'/>
-    <meta name="description"/>
-    <title>Sedona cosik</title>
-  </head>
-
-  <body>
-    <div class="wrapper"><!-- main page -->
-      <header class="page-header">
-        <nav class="main-nav">
-          <ul class="main-nav__list">
-            <li class="main-nav__item main-nav__item--active"><a href="index.html">Home</a></li>
-            <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
-            <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
-            <li class="main-nav__item">HTML Academy</li>
-          </ul>
-          <button class="main-nav__toggle main-nav__toggle--close"></button>
-          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
-        </nav>
-
-        <div calss="intro">
-          <ul class="intro-list">
-            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
-            <li class="intro-list__item intro-list__item--middle">Sedona</li>
-            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
-          </ul>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="description">
+  <title>Sedona cosik</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Sans:400,700">
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <!-- main page -->
+    <header class="page-header">
+      <nav class="main-nav">
+        <ul class="main-nav__list">
+          <li class="main-nav__item main-nav__item--active"><a href="index.html">Home</a></li>
+          <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
+          <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+          <li class="main-nav__item"><a href="#">HTML Academy</a></li>
+        </ul>
+        <div class="main-nav__toggle main-nav__toggle--close">
+          <div class="main-nav__menu-bar main-nav__menu-bar--top"></div>
+          <div class="main-nav__menu-bar main-nav__menu-bar--middle"></div>
+          <div class="main-nav__menu-bar main-nav__menu-bar--bottom"></div>
+        </div>
+        <a class="main-nav__link" href="index.html"><img class="main-nav__logo" src="images/logo.png"  alt="Sedona logo"></a>
+      </nav>
+
+      <div class="intro">
+        <div class="intro-images">
+          <h3 class="intro-images__header">Welcome to gorgeous Sedona. "Because the Grand Canyon sucks!"<h3>
+          </div>
         </div>
+        <div class="header-separator"></div>
       </header>
 
+      <!-- <section class="header-separator"> -->
```

Comment:
remove comments, if you need to save them keep them somewhere in private note. Having any commented code in production/master branch is regarded as bad manner.